product_supplierinfo_for_customer/README.rst

```diff
@@ -0,0 +1,52 @@
+.. image:: https://img.shields.io/badge/licence-AGPL--3-blue.svg
+    :alt: License: AGPL-3
+    :target: http://www.gnu.org/licenses/agpl-3.0.en.html
+
+============================================
+Use product supplier info also for customers
+============================================
+
+This modules allows to use supplier info structure, available in
+*Inventory* tab of the product form, also for defining customer information,
+allowing to define prices per customer and product.
+
+Configuration
+=============
+
+For these prices to be used in sale prices calculations, you will have
+to create a pricelist with a rule with option "Based on" with the value
+"Supplier prices on the product form" (although the text is not clear enough).
```

Comment:
Why not change the name to "Partner prices on the product form"?

Reply:
Sorry for the mistake. This text is obsolete as that options is not available in v10. Currently the prices in the pricelist are not passed to the SO. I'm wondering if adding this feature back is really needed or that can be implemented in other way :confused:

Reply:
Personally I feel it is redundant with the PL management system so I would not fight for it